pkg/acsengine/const.go

```diff
@@ -84,8 +84,8 @@ const (
 // For instance, Kubernetes release "1.7" would contain the version "1.7.2"
 var KubeConfigs = map[string]map[string]string{
 	api.KubernetesRelease1Dot7: {
-		"version":         "1.7.2",
-		"hyperkube":       "hyperkube-amd64:v1.7.2",
+		"version":         "1.7.4",
```

Comment:
@lachie83 I think what @rjtsdl is saying is that the `version` property inside this `map[string]string` is no longer used to determine the specific release version for the `1.7` channel. (Note that it's not there for `1.6` or `1.5` below.) This is an oversight, left over from when we deprecated its usage.

I think it's appropriate to use this PR to fix that! So, would you kindly delete the `"version"` string property altogether from `api.KubernetesRelease1Dot7`?